qiskit/providers/models/pulsedefaults.py

```diff
@@ -94,44 +98,335 @@ class Command(BaseModel):
     full description of the model, please check ``CommandSchema``.
 
     Attributes:
-        name (str): Pulse command name.
+        name: Pulse command name.
     """
-    def __init__(self, name, **kwargs):
+    def __init__(self, name: str, **kwargs):
         self.name = name
 
         super().__init__(**kwargs)
 
 
 @bind_schema(PulseDefaultsSchema)
 class PulseDefaults(BaseModel):
-    """Model for PulseDefaults.
-
-    Please note that this class only describes the required fields. For the
-    full description of the model, please check ``PulseDefaultsSchema``.
-
-    Attributes:
-        qubit_freq_est (list[number]): Estimated qubit frequencies in GHz.
-        meas_freq_est (list[number]): Estimated measurement cavity frequencies
-            in GHz.
-        buffer (int): Default buffer time (in units of dt) between pulses.
-        pulse_library (list[PulseLibraryItem]): Backend pulse library.
-        cmd_def (list[Command]): Backend command definition.
+    """Description of default settings for Pulse systems. These are operations or settings that
+    may be good starting points for the Pulse user. The user may modify these defaults through
+    the provided methods to build a reference to custom operations, which may in turn be used
+    for building Schedules or converting circuits to Schedules.
     """
 
-    def __init__(self, qubit_freq_est, meas_freq_est, buffer,
-                 pulse_library, cmd_def, **kwargs):
-        self.qubit_freq_est = qubit_freq_est
-        self.meas_freq_est = meas_freq_est
+    def __init__(self,
```

Comment:
This is a side-note as I know this is not the place to change. Right now the only way to produce the user level object is to provide the transport input. Really we should just be able to provide the raw variables like `CmdDef` (or whatever name we choose and the Defaults is just a container for these.